health/micro-ui/web/microplan/Dockerfile

```
@@ -1,6 +1,6 @@
-# syntax=docker/dockerfile:1.4
-FROM node:14-alpine AS deps
-WORKDIR /app
+FROM node:14-alpine3.16 AS build
+RUN apk update
+RUN apk add --no-cache 'git>2.30.0'
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)